CONTRIBUTING.md

```diff
@@ -47,7 +47,7 @@ However, participation requires adherence to fundamental ground rules:
 Software requirement: [clang-format](https://clang.llvm.org/docs/ClangFormat.html) version 12 or later.
 
 This repository consistently contains an up-to-date `.clang-format` file with rules that match the explained ones.
-For maintaining a uniform coding style, execute the command `clang-format -i *.[ch]`.
+For maintaining a uniform coding style, execute the command `clang-format -i *.[c|h]`.
```

Comment:
If you are running bash, the changed command does not work.

Reply:
should i change it to `clang-format -i *.{c,h}` or keep in the original way? For me `clang-format -i *.{c,h}` is more clear. Not sure how other people think about it